runtime/drivers/duckdb/duckdb.go

```diff
@@ -130,7 +130,20 @@ func (d Driver) Open(cfgMap map[string]any, shared bool, ac activity.Client, log
 	// Open the DB
 	err = c.reopenDB()
 	if err != nil {
-		return nil, err
+		if !strings.Contains(err.Error(), "created with an older, incompatible version of Rill") {
+			return nil, err
+		}
+		c.logger.Named("console").Info("removing db file created using old version")
+		tmpPath := cfg.DBFilePath + ".tmp"
+		_ = os.RemoveAll(tmpPath)
+		walPath := cfg.DBFilePath + ".wal"
+		_ = os.Remove(walPath)
+		_ = os.Remove(cfg.DBFilePath)
```

Comment:
As previously discussed, we should have a configProperty like `error_on_incompatible_version` and always return the error if it's true.

And the CLI should set that property to true in `rill start` if the user manually specified a DB path.

Reply:
I misunderstood our another slack conversation and thought we need to always delete the file. Will make these changes.